lib/Components/Cart/cart.html

```diff
@@ -0,0 +1,301 @@
+<!DOCTYPE html>
```

Comment:
UI suggestion - The cart page looks too white, IMO. It would be great if you can tone down the brightness by selecting a darker color or light gray.